### DISCUSSION TITLE
Mux Player controls always visible

### DISCUSSION THREAD

**Discussion by ahmedkandel** [muxinc/elements#723] [Ideas]
Hi,

Is there a way to keep the player controls and title always visible even when the user hover out?

**Comment by dylanjha** [comment]
Hi @ahmedkandel thanks for opening the issue. Mux Player does not have a way to control this yet.

I think we should be able to add it  though because the underlying library [Media Chrome](https://www.media-chrome.org/en/media-controller#autohide) has this feature with the `autohide` attribute.

We'll look into it!

**Comment by paullacour** [comment]
I also think this is a vital basic feature which should be added! Thanks
